packages/camera/camera_platform_interface/lib/src/types/media_settings.dart

```diff
@@ -0,0 +1,57 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+// ignore_for_file: avoid_equals_and_hash_code_on_mutable_classes
+
+import 'resolution_preset.dart';
+
+/// recording media settings.
```

Comment:
Could you make this more descriptive? More specifically, adding how the class is used.